ipatests/azure/templates/prepare-build.yml

```diff
@@ -2,6 +2,14 @@ steps:
 - script: |
     set -e
     sudo rm -rf /var/cache/dnf/*
+    echo "dnf.conf: enable fastestmirror"
+    sudo echo "fastestmirror = True" >> /etc/dnf/dnf.conf
```

Comment:
This line runs echo under sudo and then tries to redirect its output under the user the outer script runs on.

Reply:
I'd rather added this as a config in `ipatests/azure/` and then copied it over with `sudo cp .. ..`

Reply:
A file ``/etc/dnf/vars/fastestmirror`` with content ``enabled`` should do the trick, too.